explainability-service/manifests/opendatahub/default/trusty-deployment.yaml

```diff
@@ -0,0 +1,118 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    app.openshift.io/vcs-url: <<unknown>>
+    app.quarkus.io/commit-id: ea73183a2c1b81bd9afe31005644d374b5bdeb34
+    app.quarkus.io/build-timestamp: 2023-02-15 - 15:41:47 +0000
+    prometheus.io/scrape: "true"
+    prometheus.io/path: /q/metrics
+    prometheus.io/port: "8080"
+    prometheus.io/scheme: http
+  labels:
+    app.kubernetes.io/name: trustyai-service
+    app.kubernetes.io/version: 999-SNAPSHOT
+    app.openshift.io/runtime: quarkus
+  name: trustyai-service
+spec:
+  ports:
+    - name: http
+      port: 80
+      targetPort: 8080
+  selector:
+    app.kubernetes.io/name: trustyai-service
+    app.kubernetes.io/version: 999-SNAPSHOT
+  type: ClusterIP
+
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
```

Comment:
Do we use `ImageStream` for ODH components? I was expecting to use a fixed (aka pre-build) image from registry 🤔